static/js/containers/App.js

```diff
@@ -317,13 +321,15 @@ export default connect(state => {
   } = state
 
   const showUserMenu = dropdownMenus.has(USER_MENU_DROPDOWN)
+  const editingWidgets = state.forms[WIDGET_FORM_KEY]
```

Comment:
A couple minor things:

1. This would be more readable/accurate if you coerced this to a boolean using `R.isNil` or something like that.
1. I think this prop name should be something like `shouldShowOverlay`. The `App` component render method doesn't necessarily need to know that we want to show the overlay when editing widgets. It just needs to know whether to show the overlay or not.